staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/controller_reconcile.go

```diff
@@ -91,12 +92,29 @@ func (c *celAdmissionController) reconcilePolicyDefinition(namespace, name strin
 	}
 
 	// find GVR for params
-	paramsGVR, err := c.restMapper.RESTMapping(paramSource.GroupKind(), paramSource.Version)
+	// Parse param source into a GVK
+
+	paramSourceGV, err := schema.ParseGroupVersion(paramSource.APIVersion)
+	if err != nil {
+		// Failed to resolve. Return error so we retry again (rate limited)
+		// Save a record of this definition with an evaluator that unconditionally
+		//
+		info.configurationError = fmt.Errorf("failed to parse apiVersion of paramKind '%v'", paramSource.String())
```

Comment:
Maybe log the parse error? Right now it gets dropped so would be impossible to debug.